pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +47,47 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Get the route for service
+func GetServiceURL(service, namespace string, https bool) (string, error) {
+	urlScheme := "http://"
+	if https {
+		urlScheme = "https://"
+	}
+
+	cmdName := instanceState.Config.OcPath
+	cmdArgText := fmt.Sprintf("get route/%s", service)
+	if !isProjectExists(namespace) {
+		return "", errors.New(fmt.Sprintf("Namespace %s doesn't exits", namespace))
+	}
+	if namespace != "" {
+		cmdArgText = cmdArgText + " -n " + namespace
+	}
+
+	// json filter
+	cmdArgText += " -o json"
+	cmdOut, err := runner.Output(cmdName, strings.Split(cmdArgText, " ")...)
+	if err != nil {
+		return "", err
+	}
+
+	var route routeApi.Route
+	json.Unmarshal(cmdOut, &route)
```

Comment:
I am not sure whether extracting the host is worth this OpenShift import. I'd either extract it from the default stdout output, or if json, then I'd unmarshal into a map or create my own struct to parse this single value.